src/clients/FlexbaseClient.Company.ts

```diff
@@ -0,0 +1,38 @@
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+import IEmployee from '../models/Company/Employee';
+
+interface UserInfoPreview {
+  firstName?: string;
+  lastName?: string;
+  jobTitle?: string;
+  email?: string;
+  phone?: string;
+  id?: string;
+}
+
+export class FlexbaseClientCompany extends FlexbaseClientBase {
+  async getCompanyUsers(): Promise<IEmployee[]> {
+    try {
+      const response = await this.client
+        .url('/user')
+        .get()
+        .json<UserInfoPreview[]>();
```

Comment:
This is mostly implemented already in https://github.com/flexbase-eng/flexbase-client/blob/main/src/clients/FlexbaseClient.Person.ts#L28
It might be better to delete this class and add this function to the Person client instead